src/components/combobox/combobox.tsx

```diff
@@ -397,6 +397,16 @@ export class Combobox implements LabelableComponent, FormComponent, InteractiveC
     this.setFocus();
   };
 
+  inViewport = (): boolean => {
```

Comment:
Nitpick: this can be a class method and not an assigned function. Reasoning: no need for a bound function here.